build/common.sh

```diff
@@ -569,6 +578,30 @@ function kube::release::package_server_tarballs() {
   done
 }
 
+# This will take binaries that run on master and creates Docker images
+# that wrap the binary in them. (One docker image per binary)
+function kube::release::create_docker_images_for_master() {
```

Comment:
nit: create_docker_images_for_wrapped_binaries() (since they don't necessarily have to be master, but are for now)